File: 2024/java/junior/Junior2.java
import java.util.*;

public class Junior2 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int dusa = in.nextInt();

        while (true) {
            int yobis = in.nextInt();
            if (dusa > yobis) {
                dusa += yobis;
            } else {
                break;
            }
        }

        System.out.println(dusa);
    }
}

File: 2024/cpp/junior/junior2.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    int dusa;
    cin >> dusa;

    while (true) {
        int yobis;
        cin >> yobis;
        if (dusa > yobis) {
            dusa += yobis;
        } else {
            break;
        }
    }

    cout << dusa << endl;
}